webapp/src/components/App/Singlestudy/explore/Configuration/General/dialogs/ScenarioPlaylistDialog/index.tsx

```diff
@@ -98,7 +100,7 @@ function ScenarioPlaylistDialog({ study, open, onClose }: Props) {
       (acc, [_, { status, weight }]) => {
         if (status) {
           acc.selected += 1;
-          acc.sumWeights += weight;
+          acc.sumWeights = new Decimal(acc.sumWeights).plus(weight).toNumber();
```

Comment:
you can add a comment to explain the issue

Reply:
Ok